src/transformers/models/fastspeech2/configuration_fastspeech2.py

```diff
@@ -0,0 +1,154 @@
+        encoder_layers (`int`, *optional*, defaults to 4):
+            Number of hidden layers in the FastSpeech2 encoder.
+        decoder_layers (`int`, *optional*, defaults to 4):
+            Number of hidden layers in the FastSpeech2 decoder.
+        encoder_attention_heads (`int`, *optional*, defaults to 2):
+            Number of attention heads for each attention layer in the FastSpeech2 encoder.
+        decoder_attention_heads (`int`, *optional*, defaults to 2):
+            Number of attention heads for each attention layer in the FastSpeech2 decoder.
+        intermediate_size (`int`, *optional*, defaults to 3072):
+            Dimension of the "intermediate" (i.e., feed-forward) layer in the Transformer encoder.
+        hidden_dropout_prob (`float`, *optional*, defaults to 0.1):
+            The dropout probabilitiy for all fully connected layers in the embeddings, encoder, and pooler.
+        attention_probs_dropout_prob (`float`, *optional*, defaults to 0.1):
+            The dropout ratio for the attention probabilities.
+        max_source_positions (`int`, *optional*, defaults to 1024):
+            The maximum sequence length that this model might ever be used with. Typically set this to something large
+            just in case (e.g., 512 or 1024 or 2048).
+        initializer_range (`float`, *optional*, defaults to 0.02):
+            The standard deviation of the truncated_normal_initializer for initializing all weight matrices.
+        Example:
+
+    ```python
+    >>> from transformers import FastSpeech2Model, FastSpeech2Config
+
+    >>> # Initializing a FastSpeech2 fastspeech2 style configuration
+    >>> configuration = FastSpeech2Config()
+
+    >>> # Initializing a model from the fastspeech2 style configuration
+    >>> model = FastSpeech2Model(configuration)
+
+    >>> # Accessing the model configuration
+    >>> configuration = model.config
+    ```"""
+    model_type = "fastspeech2"
+
+    def __init__(
+        self,
+        n_frames_per_step=1,
+        output_frame_dim=80,
+        encoder_embed_dim=256,
+        speaker_embed_dim=64,
+        dropout=0.2,
+        max_source_positions=1024,
+        encoder_attention_heads=2,
+        fft_hidden_dim=1024,
+        fft_kernel_size=9,
+        attention_dropout=0,
+        encoder_layers=4,
+        decoder_embed_dim=256,
+        decoder_attention_heads=2,
+        decoder_layers=4,
+        add_postnet=False,
+        postnet_conv_dim=512,
+        postnet_conv_kernel_size=5,
+        postnet_layers=5,
+        postnet_dropout=0.5,
+        vocab_size=75,
+        num_speakers=1,
+        var_pred_n_bins=256,
+        var_pred_hidden_dim=256,
+        var_pred_kernel_size=3,
+        var_pred_dropout=0.5,
+        pitch_max=5.733940816898645,
+        pitch_min=-4.660287183665281,
+        energy_max=3.2244551181793213,
+        energy_min=-4.9544901847839355,
+        initializer_range=0.0625,
+        mean=True,
```

Comment:
not a huge fan of the naming here `mean` and `std` don't sound like flag variables here.

Reply:
Yes, I will update them with `use_mean` and `use_standard_deviation`!